ProjectSource/app-library/src/test/java/com/kayako/sdk/android/k5/kre/base/KreConnectionFactoryTest.java

```diff
@@ -0,0 +1,24 @@
+package com.kayako.sdk.android.k5.kre.base;
+
+import static org.hamcrest.CoreMatchers.notNullValue;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ErrorCollector;
+
+public class KreConnectionFactoryTest {
+
+    @Rule
+    public final ErrorCollector errorCollector = new ErrorCollector();
```

Comment:
✅ Nice to see JUnit ErrorCollector rule at work.

This way the developer can assert units-of-work without running tests again and again, after each failure in a @Test.

**No change suggested, looks good!** 
  

<a id="aa4ecef0-4946-11e8-8af6-c1e484fb2340" href="none" codereview-uuid-reserved-tag />